src/ingest_validation_tests/tiff_validator.py

```diff
@@ -0,0 +1,38 @@
+from os import cpu_count
+from multiprocessing import Pool
+from typing import List
+
+import tifffile
+from ingest_validation_tools.plugin_validator import Validator
+
+def _log(message: str):
+    print(message)
+
+
+def _check_tiff_file(path: str) -> str or None:
+    try:
+        with tifffile.TiffFile(path) as tfile:
+            for page in tfile.pages:
+                _ = page.shape
+                _ = page.dtype
+            return None
+    except Exception as excp:
+        _log(f"{path} is not a valid TIFF file: {excp}")
+        return f"{path} is not a valid TIFF file"
+
+
+class TiffValidator(Validator):
+    description = "Recursively test all tiff files that are not ome.tiffs for validity"
+    cost = 1.0
+    def collect_errors(self, **kwargs) -> List[str]:
+        threads = kwargs.get('coreuse', None) or cpu_count() // 4 or 1
```

Comment:
I think 'or 1' is redundant here since // 4 will return an integer and the possible minimum for that will be 1 (every computer has at least 1 cpu and 1 // 4 is still 1)

Reply:
Ah, but 1 // 4 is not 1!
```
$ python
Python 3.9.16 (main, Mar  8 2023, 14:00:05) 
[GCC 11.2.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 1 // 4
0
```